qa/tasks/ceph2.py

```diff
@@ -724,7 +709,9 @@ def restart(ctx, config):
             healthy(ctx=ctx, config=dict(cluster=cluster))
     if config.get('wait-for-osds-up', False):
         for cluster in clusters:
-            wait_for_osds_up(ctx=ctx, config=dict(cluster=cluster))
+            # FIXME this method is not defined!!!
+            # wait_for_osds_up(ctx=ctx, config=dict(cluster=cluster))
```

Comment:
@sage: *`wait_for_osds_up`: FIXME this method is not defined!!!*

Reply:
should be
```
            ctx.managers[cluster].wait_for_all_osds_up()
```

Reply:
done